src/JSInterop/Microsoft.JSInterop/test/JSInProcessRuntimeTest.cs

```diff
@@ -111,7 +111,7 @@ public class InvokeArgs
                 public string? ArgsJson { get; set; }
             }
 
-            protected override void BeginInvokeJS(long asyncHandle, string identifier, string? argsJson, JSCallResultType resultType, long targetInstanceId)
+            protected override void BeginInvokeJS(long asyncHandle, string identifier, string? argsJson, byte[][]? byteArrays, JSCallResultType resultType, long targetInstanceId)
```

Comment:
Everywhere that we have a pair like `argsJson` and `byteArrays`, it would be great to rename the latter to something like `argsByteArrays` to clarify that it's part of the same logical concept (the args).

Reply:
Though since we're doing breaking changes, we might even want to define something like:

```cs
public readonly struct InvocationArgs
{
    public readonly string JsonData { get; }
    public readonly byte[][] BinaryData { get; }
}
```

Reply:
https://github.com/dotnet/aspnetcore/pull/32259/commits/b0b5a6bab7c12a4b117e4d62d9dcac8e02334fe3